holmes/plugins/toolsets/__init__.py

```
@@ -54,7 +56,9 @@
     return file_toolsets
 
 
-def load_python_toolsets(dal: Optional[SupabaseDal]) -> List[Toolset]:
+def load_python_toolsets(
+    dal: Optional[SupabaseDal], use_legacy_k8s_logs: bool = False
```

Ruff (0.8.2)

Boolean-typed positional argument in function definition

(FBT001)